float64 neck_rotation_theta  # 颈旋转关节角度
float64 neck_front_swing_theta # 颈前摆关节角度
bool is_find_state # 是否找球状态
bool is_follow_state # 是否跟球状态